voxblox_tango_interface/test/mesh_tango_tsdf.cc

```diff
@@ -21,8 +21,11 @@ int main(int argc, char** argv) {
   TangoLayerInterface::Ptr layer_from_file;
   io::TangoLoadLayer(file, &layer_from_file);
 
-  std::cout << "Layer memory size: " << layer_from_file->getMemorySize()
-            << "\n";
+  LOG(WARNING) << "Layer memory size: " << layer_from_file->getMemorySize()
```

Comment:
this seems like a debugging output. Use LOG(INFO) or VLOG.

Reply:
👍